src/shared/ui/WriteToUsIcon/index.tsx

```diff
@@ -0,0 +1,2 @@
+import WriteToUsIcon from "./WriteToUsIcon";
+export default WriteToUsIcon;
```

Comment:
if you get extra files in the folder? 
I suggest using aggregate export example: export * from './fileName'.